book/security.rst

```diff
@@ -1043,6 +1043,17 @@ if your User object has a ``getFirstName()`` method, you could use that::
         return new Response('Well hi there '.$user->getFirstName());
     }
 
+.. versionadded:: 2.8
+     The ``security.current_user_provider`` service was introduced in Symfony 2.8. It makes it 
+     easier to retrieve the current logged in user from a service. 
+     
+    .. code-block:: php
+
+        $user = $this->container->get('security.current_user_provider')->getUser();
+        if ($user === null) {
```

Comment:
I have two comments about this line:

1. Should we use in the docs the same conditional style as in the code (`if (null === $user)`)?
2. In the docs we explain that in Symfony applications, even anonymous users are technically logged in. Not only that, but they are even authenticated ( `IS_AUTHENTICATED_ANONYMOUSLY`). That's why I find strange that the user can be `null` here. I expect to get an anonymous user.